controller/hybridgateway/converter/http_route.go

```diff
@@ -312,13 +490,35 @@ func (c *httpRouteConverter) translate(ctx context.Context, logger logr.Logger)
 					WithOwner(c.route).
 					WithRouteRef(routeName).Build()
 				if err != nil {
+					log.Error(logger, err, "Failed to build KongPluginBinding resource, skipping binding",
+						"binding", bindingName,
+						"plugin", pluginName,
+						"kongRoute", routeName)
+					translationErrors = append(translationErrors, fmt.Errorf("failed to build KongPluginBinding %s: %w", bindingName, err))
 					continue
 				}
 				c.outputStore = append(c.outputStore, &binding)
+
+				log.Trace(logger, "Successfully built KongPlugin and KongPluginBinding resources",
```

Comment:
We use debug level elsewhere for successful operations
```suggestion
				log.Debug(logger, "Successfully built KongPlugin and KongPluginBinding resources",
```